source/index.html

```diff
@@ -1,77 +1,143 @@
+                <img src="./assets/gloves.png" alt="">
+                <h2>We have you covered!</h2>
+                <p>No matter the weather.</p>
+            </div>
+            <div class="alpaca_sweater">
+                <img src="./assets/DT_marquette.png" alt="">
+                <h2>In historic downtown Marquette, MI</h2>
+                <p>Pure Michigan at its best.</p>
+            </div>
+            <div class="relaxed">
+                <img src="./assets/relax-enviroment.png" alt="">
+                <h2>Relaxing enviroment!</h2>
+                <p>Come in and shop in comfort.</p>
+            </div>
+        </div>
+    </section>
+
+    <img class="img-divider" src="./assets/landscape-1.png" alt="">
+
+    <section id="shopping">
+        <h1>Shopping</h1>
+        <h3>Can&apos;t get into the store? You can find most of your favorites like incense and body oils in our online store.</h3>
+        <p><a href="https://squareup.com/store/panara-imports">Online Store</a></p>
+        <p><a href="http://fashionrevolution.org/">Fashion Revolution</a></p>
+        <div class="container">
+            <div class="panara-shirt">
+                <img src="./assets/panara-shirt.png" alt="">
+                <h2>Panara merchandise</h2>
+                <p>Great in any weather</p>
             </div>
-            <div class="content-box">
-                <a href="http://goo.gl/maps/7xS1a">
-                    <img src="./images/panara-imports-static-map.png" alt="" width="300" height="300">
-                </a>
+            <div class="black-jacket">
+                <img src="./assets/black-jacket.png" alt="">
+                <h2>Cloths and accesories for any occasion</h2>
+                <p>Check daily for updates and new products</p>
             </div>
-        </main>
-    </body>
+            <div class="oils">
+                <img src="./assets/oils.png" alt="">
+                <h2>Essential oils and Incense</h2>
+                <p>Available year round in store or online</p>
+            </div>
+        </div>
+    </section>
+
+    <img class="img-divider" src="./assets/landscape-2.png" alt="">
+
+    <section id="blog">
+        <h1>Blog</h1>
+        <h3>My life, my store, my passions and random observations.</h3>
+        <div>
+            <div class="blog-post-1">
+                <img src="./assets/emma.png" alt="">
+                <h4><a href="http://panaraimports.tumblr.com/">Persons of Interest</a></h4>
+                <p>There are a lot of people in the world that inspire and represent what Panara is about, a world of different. I find celebrities and locals alike who follow their dreams, push boundaries and always fight for what is best for their fellow man.</p>
+            </div>
+            <div class="blog-post-2">
+                <img src="./assets/great_lakes_jewlery.png" alt="">
+                <h4><a href="http://panaraimports.tumblr.com/">Products of Interest</a></h4>
```

Comment:
Same here … what would make me think this is a link, and why would I expect these words to link to the blog. Will the blog have products in it? Could be pretty confusing for users.